packages/flutter_tools/lib/src/commands/init.dart

```diff
@@ -201,14 +201,27 @@ void main() {
   runApp(
     new MaterialApp(
       title: "Flutter Demo",
-      routes: {
+      routes: <String, RouteBuilder>{
         '/': (RouteArguments args) => new FlutterDemo()
       }
     )
   );
 }
 
-class FlutterDemo extends StatelessComponent {
+class FlutterDemo extends StatefulComponent {
+  @override
```

Comment:
Ah, gotcha. We want a hint when `@override` isn't used. I'd also love to see a Quick Fix (add missing @override)